x-pack/plugins/snapshot_restore/public/application/sections/home/policy_list/policy_list.tsx

```diff
@@ -157,9 +165,44 @@ export const PolicyList: React.FunctionComponent<RouteComponentProps<MatchParams
     const policySchedules = policies.map((policy: SlmPolicy) => policy.schedule);
     const hasDuplicateSchedules = policySchedules.length > new Set(policySchedules).size;
     const hasRetention = Boolean(policies.find((policy: SlmPolicy) => policy.retention));
+    const isSlmRunning = slmStatus?.operation_mode === SLM_STATE.RUNNING;
 
     content = (
       <section data-test-subj="policyList">
+        {!isSlmRunning ? (
+          <Fragment>
+            <EuiCallOut
+              title={
+                <FormattedMessage
+                  id="xpack.snapshotRestore.slmWarningTitle"
+                  defaultMessage="Snapshot lifecycle management (SLM) is not running"
+                />
+              }
+              color="warning"
+              iconType="warning"
+            >
+              <FormattedMessage
+                id="xpack.snapshotRestore.slmWarningDescription"
+                defaultMessage="Policies are not executed. You must restart SLM {slmDocLink}"
+                values={{
+                  slmDocLink: (
+                    <EuiLink
+                      href={docLinks.links.snapshotRestore.slmStart}
+                      external={true}
+                      target="_blank"
+                    >
+                      {i18n.translate('xpack.snapshotRestore.slmDocLink', {
+                        defaultMessage: 'by using the API.',
```

Comment:
`by` implies that you can make any api call to restart SLM

```suggestion
                        defaultMessage: 'using the API.',
```